types/serverless/plugins/aws/provider/awsProvider.d.ts

```diff
@@ -761,7 +761,7 @@ declare namespace Aws {
 declare class Aws {
     constructor(serverless: Serverless, options: Serverless.Options);
 
-    naming: { [key: string]: () => string };
+    naming: { [key: string]: (param: string) => string };
```

Comment:
Currently this is a required parameter:

```suggestion
    naming: { [key: string]: (param?: string) => string };
```

(sorry forgot to hit "start a review" 😅)